src/scripts/calendar.ts

```diff
@@ -99,19 +100,33 @@ function clearCalendar() {
  * 
  * @param {Event} ev - Native Event Object.
  */
-function detailsToggle(ev) {
-    if (ev.matches) {
-        $details.removeAttribute('open');
-    } else {
-        $details.setAttribute('open', '');
+function detailsToggle(ev: MediaQueryListEvent | MediaQueryList): void {
+    const $details: HTMLDetailsElement | null = document.querySelector('details');
```

Comment:
We shouldn't need to restate the `$details` element here, it's defined at the top of the file.

Reply:
Removed. Also inverted the guard clause for null check.